app/views/produce_items/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <h1 class="page__title"> <%= I18n.t('produce_item.index.title') %> </h1>
 
 <table class="produce-table">
-    <% @produce_items.find_each do |produce|  %>
+    <% @produce_items.each do |produce|  %>
```

Comment:
**question:** why change from `find_each` to `each`?

Reply:
Sorry, I was experimenting with the sort and left this behind. If I'm not mistaken, I couldn't sort alphabetically with the find_each.

Reply:
That is correct. This might become a problem as the database of items grows larger, but lets keep `each` for the time being, since it makes no different and alphabetic order makes more sense.

Reply:
(disclosure why changing ordering breaks `find_each`: it is because find_each relies on IDs for batching records)